consensus/polybft/consensus_runtime.go

```diff
@@ -394,6 +411,31 @@ func (c *consensusRuntime) FSM() error {
 	return nil
 }
 
+// getNumberOfIteration returns number of iteration that are needed for the proposer calculation
+func getNumberOfIteration(parent *types.Header, epochNumber uint64, c *consensusRuntime) (uint64, error) {
```

Comment:
Change this to method of `consensusRuntime`, since it is required as a parameter?

Reply:
Everything regrading this thing will be done through: https://polygon.atlassian.net/browse/EVM-205